### DISCUSSION TITLE
How to post/subpost

### DISCUSSION THREAD

**Discussion by HelloItsDev** [arthelokyo/astrowind#201] [Q&A]
Is there a way to create subposts in markdown, so that the subpost is a child article of a post and therefore is accessible at the url post/subpost?

**Comment by widgeter** [comment]
Hi @HelloItsDev 

We had accidentally disabled support for sub posts trying to build the bases to allow posts in different languages ​​(i18n).
A small adjustment has been made to `utils/blogs.ts` and now you can have sub post: https://github.com/onwidget/astrowind/commit/3ef024f14982556367cdb3b1b19b5cc580e6b819

For this we recommend to structure the posts in which you want to have subposts as folders in src/content, example:

```
/
├── content/
│   ├── post/
│   │   ├── post-without-sub-post-1.md
│   │   ├── post-without-sub-post-2.md
│   │   ├── post-with-sub-post/
│   │   │   ├── index.md
│   │   │   ├── sub-post-1.md
│   │   │   └── sub-post-1.md
```

In this example, `post-with-sub-post/index.md` is the parent post, `post-with-sub-post/sub-post-1.md` and `post-with-sub-post/sub-post-2.md` the sub posts

---

QUESTION: Something I don't know about right now and I would like to know your suggestion is what is the best way to show posts (in /blog) when there are also sub posts in the blog. If it is preferable to only show the parent posts in /blog or it is good that the parent posts and their children appear as well.
